app/models/question.rb

```diff
@@ -1,3 +1,12 @@
 class Question < ActiveRecord::Base
   has_many :answers
+
+
+    def display_resolved
```

Comment:
That makes sense. And actually, because I was indecisive I didn't realize I had this logic duplicated in the view and in the model :grin: 

Now, would it be just as viable to wrap the conditional that's currently in the view in a method definition inside of the controller? Or does that just bring us back to a dependency that we don't really need?